packages/mui-material/src/InitColorSchemeScript/InitColorSchemeScript.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
```

Comment:
The change in this file is required to generate the API page.
All of the behavior remains the same.